presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -192,6 +204,36 @@ public Duration getElapsedTime()
     }
 
     @JsonProperty
+    public Duration getExecutionTime()
+    {
+        return executionTime.convertToMostSuccinctTimeUnit();
+    }
+
+    @JsonProperty
+    public Duration getCpuTime()
+    {
+        return cpuTime;
+    }
+
+    @JsonProperty
+    public long getExecutionTimeMillis()
```

Comment:
I'm a bit confused. Why do we need both for backwards compatibility? Both of these methods are new